src/components/Chat/Chat.tsx

```diff
@@ -159,8 +161,16 @@ export const Chat = memo(() => {
           }
           return !modelIds.includes(conv.model.id);
         });
-    setIsNotAllowedModel(isNotAllowed);
-  }, [selectedConversations, models, modelsIsLoading]);
+    if (isNotAllowed) {
+      setNotAllowedType(EntityType.Model);
+    } else if (
+      selectedConversations.some((conversation) =>
+        conversation.selectedAddons.some((addonId) => !addonsMap[addonId]),
+      )
+    ) {
+      setNotAllowedType(EntityType.Addon);
+    } else setNotAllowedType(null);
```

Comment:
nit: let's wrap this setNotAllowedType(null) in brackets because it will simplify understanding of this block 
`
if () {
...
} else if () {
...
} else {
...
}
`